doc/source/index.rst

```diff
@@ -37,12 +37,33 @@ Using the theme
 
 #. Set the options to link to the git repository and bug tracker.
 
+    ``otcdocs_git_fqdn``
+      FQDN of the used git hosting. It is used to construct page
+      edit/source/bug links. `github.com` is a default value.
+
    ``otcdocs_repo_name``
```

Comment:
Something is wrong here, look at the preview: https://github.com/opentelekomcloud/otcdocstheme/blob/19060d309b59b67a7c2ea1e3a836a4d743ffbeb3/doc/source/index.rst

Reply:
done